account_operating_unit/tests/test_payment_operating_unit.py

```diff
@@ -3,7 +3,7 @@
 # © 2016 Serpent Consulting Services Pvt. Ltd.
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 from odoo.addons.account_operating_unit.tests import\
-    test_account_operating_unit as test_ou
+    test_account_operating_unit as test_ou   # noqa
```

Comment:
Is it complaining about indentation? If yes using parenthesis is nicer than the backslash

```python
from odoo.addons.account_operating_unit.tests import (
    test_account_operating_unit as test_ou
)
```

Reply:
Actually it is complaining about the relative import. It is asking for an absolute import like `from . import ...`

Anyway i'll put the brackets it's nicer. Thanks